runtime-light/crypto/crypto_schema.tl

```diff
@@ -0,0 +1,34 @@
+---types---
+
+// Boolean
+boolFalse#bc799737 = Bool;
```

Comment:
Why do we need to define basic TL types one more time?

Reply:
For now this TL-schema is self-sufficient, all common types are copy-pasted here. It's parsed by hands and it's easier to have the single source of truth

Reply:
How does it makes life harder?
Which problems we are gonna to face?
It doesn't use codegen yet, so there is the same file in `k2` repo, I think we should copy-paste primitives till the merging to the one schema